modules/machineset-creating-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,292 @@
+            lun: 0
+            diskSizeGB: 4
+            deletionPolicy: Delete
+            cachingType: None
+            managedDisk:
+              storageAccountType: UltraSSD_LRS
+endif::mapi[]
+             ...
+----
++
+<1> Specify a label to use to select a node that is created by this machine set. This procedure uses `disk.ultrassd` for this value.
+<2> These lines enable the use of ultra disks.
+ifdef::mapi[For `dataDisks`, include the entire stanza.]
+
+. Create a machine set using the updated configuration by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <machine-set-name>.yaml
+----
+
+ifdef::pvc[]
+. Create a storage class that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ultra-disk-sc <1>
+parameters:
+  cachingMode: None
+  diskIopsReadWrite: "2000" <2>
+  diskMbpsReadWrite: "320" <3>
+  kind: managed
+  skuname: UltraSSD_LRS
+provisioner: disk.csi.azure.com <4>
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer <5>
+----
+<1> Specify the name of the storage class. This procedure uses `ultra-disk-sc` for this value.
+<2> Specify the number of IOPS for the storage class.
+<3> Specify the throughput in MBps for the storage class.
+<4> For Azure Kubernetes Service (AKS) version 1.21 or later, use `disk.csi.azure.com`. For earlier versions of AKS, use `kubernetes.io/azure-disk`.
+<5> Optional: Specify this parameter to wait for the creation of the pod that will use the disk.
+
+. Create a persistent volume claim (PVC) to reference the `ultra-disk-sc` storage class that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ultra-disk <1>
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: ultra-disk-sc <2>
+  resources:
+    requests:
+      storage: 4Gi <3>
+----
+<1> Specify the name of the PVC. This procedure uses `ultra-disk` for this value.
+<2> This PVC references the `ultra-disk-sc` storage class.
+<3> Specify the size for the storage class. The minimum is 4Gi.
+
+. Create a pod that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-ultra
+spec:
+  nodeSelector:
+    disk: ultrassd <1>
+  containers:
+  - name: nginx-ultra
+    image: alpine:latest
+    command:
+      - "sleep"
+      - "infinity"
+    volumeMounts:
+    - mountPath: "/mnt/azure"
+      name: volume
+  volumes:
+    - name: volume
+      persistentVolumeClaim:
+        claimName: ultra-disk <2>
+----
+<1> Specify the label of the machine set that enables the use of ultra disks. This procedure uses `disk.ultrassd` for this value.
+<2> This pod references the `ultra-disk` PVC.
+endif::pvc[]
+
+.Verification
+
+#Would prefer to replace the two `oc` steps here with something from the web console. Otherwise, need some help understanding the `oc debug` step.#
```

Comment:
Need some info here
Cc @damdo

Reply:
`oc debug node/<node-name> -- <command>` is the recommended way in which users can get a shell into an OpenShift Node. Since `oc debug` will start a debugging container with a shell on the Node rather than giving you the actual Node shell, you won't be using the actual underlying host OS binaries. With the `chroot /host` command we can get to access those binaries and execute the `lsblk` binary available on the host OS. This way we can see the block devices attached to the host OS VM.